src/common/types.ts

```diff
@@ -83,17 +83,7 @@ export interface KeyboardDataWithTime extends KeyboardMetadata {
  * Relevant metadata for a particular project.
  *
  */
-export interface ProjectMetadata {
-  /**
-   * The name of language. Could be different than what is fetched by an API.
-   */
-  langName: string;
-
-  /**
-   * The valid BCP-47 tag for the language.
-   */
-  bcp47Tag: string;
-
+export interface ProjectMetadata extends KeyboardMetadata {
```

Comment:
I think the reason this used to be like this is because it matched the Keyman APIs, but their exact APIs are not really quite suited for _this_ app. Thanks for cleaning it up!